Documentation/articles/getting_started/getting_started.md

```diff
@@ -21,3 +21,7 @@ By the end of this tutorial set, you will have a working project to start buildi
 ## Migrating from MonoGame 3.7 to 3.8
 
 If you currently have a MonoGame 3.7 project and wish to upgrade it to MonoGame 3.8, please check our handy [Upgrade Guide](~/articles/migrate38.md).
+
+## Migrating from XNA to MonoGame
```

Comment:
I would argue to removing "to MonoGame" from this.  It isn't needed.  Same with the section above "Migrating from 3.7".

@Jjagg ?

Reply:
Agreed on this one, other one I like MG 3.7 better, but don't have a strong opinion on it.